internal/config/cgroupmanager/cgroupmanager.go

```diff
@@ -0,0 +1,69 @@
+// +build linux
+
+package cgroupmanager
+
+import (
+	"github.com/cri-o/cri-o/internal/config/node"
+	"github.com/pkg/errors"
+)
+
+const (
+	cgroupMemorySubsystemMountPathV1 = "/sys/fs/cgroup/memory"
+	cgroupMemorySubsystemMountPathV2 = "/sys/fs/cgroup"
+	scopePrefix                      = "crio"
+	// minMemoryLimit is the minimum memory that must be set for a container.
+	// A lower value would result in the container failing to start.
+	minMemoryLimit = 12582912
+	// CgroupfsCgroupManager represents cgroupfs native cgroup manager
+	CgroupfsCgroupManager = "cgroupfs"
+	// SystemdCgroupManager represents systemd native cgroup manager
+	SystemdCgroupManager = "systemd"
+
+	DefaultCgroupManager = SystemdCgroupManager
```

Comment:
Can we make those consts private and only the struct types public? The consts should be accessible via the `Name()` method, right?